src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,29 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    private TaskList taskManager = new TaskList();
+
+    /**
+     * You should have your own function to generate a response to user input.
+     * Replace this stub with your completed method.
+     */
```

Comment:
I think this does not follow the JavaDoc conventions because it lacks a @return parameter, but I believe this is something might have missed by accident, because many other methods have the proper JavaDocs documentation.

Reply:
Thank you for the feedback, will look to resolve this asap